android/KMEA/app/src/test/java/com/tavultesoft/kmea/logic/ResourceUpdateToolTest.java

```diff
@@ -0,0 +1,72 @@
+package com.tavultesoft.kmea.logic;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import androidx.test.core.app.ApplicationProvider;
+
+import com.tavultesoft.kmea.R;
+
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+@RunWith(RobolectricTestRunner.class)
+public class ResourceUpdateToolTest {
```

Comment:
Minor thing. Can you rename the file and class to `ResourcesUpdateToolTest` (plural Resources) since it's testing `ResourcesUpdateTool`?